<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>detaileditnote</class>
 <widget class="QDialog" name="detaileditnote">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detail Edit Note</string>
  </property>
  <widget class="QTextEdit" name="title_textEdit_detaileditnote">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>20</y>
     <width>501</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="content_textEdit_detaileditnote">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>591</width>
     <height>261</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton_detaileditnote">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Speichern</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton_detaileditnote">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>380</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Zur√ºck</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_t_detaileditnote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Titel: </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_c_detaileditnote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Inhalt:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
